docs/intro/tutorial.rst

```diff
@@ -287,8 +287,8 @@ to be scraped, you can at least get **some** data.
 
 Besides the :meth:`~scrapy.selector.SelectorList.getall` and
 :meth:`~scrapy.selector.SelectorList.get` methods, you can also use
-the :meth:`~scrapy.selector.SelectorList.re` method to extract using `regular
-expressions`_:
+the :meth:`~scrapy.selector.SelectorList.re` method to extract using
+:mod:`regular expressions <python:re>`:
```

Comment:
The issue with this is that `regular expressions` gets rendered as a module, `regular expressions`, which is not intended. You need to either find a way to link without :mod:, or rewrite it in some other way (e.g. “using Python’s re module”, “using regular expressions (re)”, or similar).

Same applies to dictionary-like and similar scenarios.

Reply:
I looked into intersphinx python's inventory and found links following pattern `https://docs.python.org/3/library/*` can be linked using `:doc:` tags. I have done these changes and updated my last commit.